backend/src/game/player.py

```diff
@@ -14,6 +14,7 @@ def __init__(self):
         self.maximum_health = constants.DEFAULT_HEALTH
         self.current_health = self.maximum_health
         self.location = Locations.OUTSIDE
+        self.is_alive = bool(0)
```

Comment:
bool(0) is false isn't it? so init a player as not alive? seems backwards

Reply:
Derpity derp derp